web/src/gql/provider.tsx

```diff
@@ -44,7 +49,7 @@ const Provider: React.FC<Props> = ({ children }) => {
 
   const client = new ApolloClient({
     uri: endpoint,
-    link: ApolloLink.from([errorLink, authLink, httpLink]),
+    link: ApolloLink.from([errorLink, authLink, uploadLink, httpLink]),
```

Comment:
(Just learnt this recently) Apparently you don't need httpLink if you have uploadLink.
So please uninstall httpLink and remove from here.

Also, after you do that, if you have an issue with the createUploadLink type being wrong, you'll have to downgrade the @type file to 17.0.0.
If you have no issues, please let me know, because maybe apollo-client's newest version fixed a type issue.

Reply:
Done!
I removed httpLink and I didn't get any type error. maybe apollo-client's newest version fixed the type issue.